### PR TITLE
update events

### PR DESCRIPTION
**Pull request by kjroelke**

### CODE REVIEW

**Review by blake-perkins-dev — CHANGES_REQUESTED** [review]
https://choctawlandev.wpengine.com/things-to-do and https://choctawlandstg.wpengine.com/things-to-do - both don't have an Events feed on the page to review (they were in draft). I undrafted them and none have images, so I can't review this.

**Comment by kjroelke** [comment]
Well that sucks. 

I'll copy the env down

**Comment by blake-perkins-dev** [comment]
> Well that sucks.
> 
> I'll copy the env down

Even with adding an image, I can't see anything on the page. Just a green line
Screenshot 2024-12-05 at 12 10 54 PM

**Comment by kjroelke** [comment]
@blake-perkins-dev ready for review on https://choctawlandstg.wpengine.com/events

**Comment by blake-perkins-dev** [comment]
I know this is going to be the most ridiculous feedback. Can you make the aspect ratio 33/50? This seems to be what it's using for most of the images.

Please use this for both the Events and the Things To Do page.

File: wp-content/themes/choctaw-landing/src/styles/components/_event-preview.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.event-preview {
+	img {
+		aspect-ratio: 33 / 50;
```
Comment (blake-perkins-dev) [inline wp-content/themes/choctaw-landing/src/styles/components/_event-preview.scss:3]: Noice! 👌

**Review by blake-perkins-dev — APPROVED** [review]
This looks so good! 10/10

**Pull request merged**